lib/sycamore/sycamore/docset.py

```diff
@@ -26,7 +26,7 @@
 
 class DocSet:
     """
-    A DocSet, short for “documentation set,” is a distributed collection of documents bundled together for processing.
+    A DocSet, short for “documentation set”, is a distributed collection of documents bundled together for processing.
```

Comment:
DocSet is short for Document Set I'm pretty sure

Reply:
changed